charts/flux-instance/templates/instance.yaml

```diff
@@ -37,6 +37,9 @@ spec:
   {{- if .Values.instance.sync.url }}
   sync:
     kind: {{ .Values.instance.sync.kind }}
+    {{- if .Values.instance.sync.interval }}
+    interval: {{ .Values.instance.sync.interval }}
+    {{- end }}
```

Comment:
Let's use the default from the CRD which is `1m`, we can remove this `if` here and set in values the default.

Reply:
I thought it'd be best to allow the CRD to control the default, but sure, no problem